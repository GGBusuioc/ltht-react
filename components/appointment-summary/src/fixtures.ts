import { Encounter, EncounterStatusCode, PartialDateTimeKindCode, Metadata } from '@ltht-react/types'

const mockMetadata: Metadata = {
  dataSources: [
    {
      display: 'Mock',
    },
  ],
}

const AppointmentOne: Encounter = {
  id: '346ca927-67e0-e931-a2c7-115087226fg2|appointment',
  metadata: mockMetadata,
  type: [
    {
      text: 'See GP regarding sore throat',
      coding: [{ display: 'GP', code: '64706' }],
    },
  ],
  period: {
    start: {
      value: '2020-04-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  serviceProvider: {
    display: 'Beeston Village Surgery',
    typeName: 'GP',
  },
  status: EncounterStatusCode.Planned,
}

const AppointmentTwo: Encounter = {
  id: '941ca927-34e0-e911-a2c7-005087226fe4|appointment',
  metadata: mockMetadata,
  type: [
    {
      text: 'Chest x-ray for recent car accident',
      coding: [{ display: 'Hospital', code: '26706' }],
    },
  ],
  period: {
    start: {
      value: '2019-12-12T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  serviceProvider: {
    display: 'Chapel Allerton Hospital',
    typeName: 'Hospital',
  },
  status: EncounterStatusCode.Finished,
}

export { AppointmentOne, AppointmentTwo }
