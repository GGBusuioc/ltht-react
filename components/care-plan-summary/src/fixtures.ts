import { CarePlan, CarePlanStatusCode, PartialDateTimeKindCode, CarePlanIntentCode, Metadata } from '@ltht-react/types'

const mockMetadata: Metadata = {
  dataSources: [
    {
      display: 'Mock',
    },
  ],
}

const CarePlanOne: CarePlan = {
  id: '580ca927-34e0-e911-a2c7-005056926fe4|careplan',
  metadata: mockMetadata,
  author: { display: 'Joe Bloggs', type: 'Test' },
  intent: CarePlanIntentCode.Plan,
  period: {
    start: {
      value: '2013-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
    end: {
      value: '2014-02-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  status: CarePlanStatusCode.Active,
  title: 'Test Care Plan Summary One',
}

const CarePlanTwo: CarePlan = {
  id: '690ca927-68e0-c511-b2c7-898056926ze4|careplan',
  metadata: mockMetadata,
  author: { display: 'John Mclane', type: 'Test' },
  intent: CarePlanIntentCode.Proposal,
  period: {
    start: {
      value: '2014-02-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
    end: {
      value: '2015-03-04T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  status: CarePlanStatusCode.Completed,
  title: 'Test Care Plan Summary Two',
}

export { CarePlanOne, CarePlanTwo }
