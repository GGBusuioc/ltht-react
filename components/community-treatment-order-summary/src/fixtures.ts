import {
  LypftCommunityTreatmentOrder,
  CarePlanStatusCode,
  PartialDateTimeKindCode,
  CarePlanIntentCode,
  Metadata,
} from '@ltht-react/types'

const mockMetadata: Metadata = {
  dataSources: [
    {
      display: 'Mock',
    },
  ],
}

const CommunityTreatmentOrderOne: LypftCommunityTreatmentOrder = {
  id: '580ca927-34e0-e911-a2c7-005056926fe4|communityTreatmentOrder',
  metadata: mockMetadata,
  consentToTreat: 'Consent Not Given',
  legalStatus: {
    coding: [{ display: 'Approved', code: '3309007' }],
    text: 'Approved',
  },
  period: {
    start: {
      value: '2013-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
    end: {
      value: '2014-02-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  restrictions: 'No restrictions',
}

const CommunityTreatmentOrderTwo: LypftCommunityTreatmentOrder = {
  id: '690ca927-68e0-c511-b2c7-898056926ze4|communityTreatmentOrder',
  metadata: mockMetadata,
  consentToTreat: 'Consent Given',
  legalStatus: {
    coding: [{ display: 'Pending', code: '779007' }],
    text: 'Pending',
  },
  period: {
    start: {
      value: '2020-01-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  restrictions: 'No Restrictions',
}

export { CommunityTreatmentOrderOne, CommunityTreatmentOrderTwo }
