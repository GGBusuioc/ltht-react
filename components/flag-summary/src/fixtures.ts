import { Flag, FlagStatusCode, PartialDateTimeKindCode, Metadata } from '@ltht-react/types'

const mockMetadata: Metadata = {
  dataSources: [
    {
      display: 'Mock',
    },
  ],
}

const FlagOne: Flag = {
  id: '580ca927-34e0-e911-a2c7-005056926fe4|personalertandhazard',
  metadata: mockMetadata,
  author: {
    display: 'John Smith',
    reference:
      'https://phoenixqa.careworks.ie/Services/api/fhir/v3/practitioner/9c1ad2c0-ec34-e911-a2c5-005056926fe4|systemuser',
    type: 'Practitioner',
    identifier: undefined,
  },
  code: {
    coding: [{ display: 'Lives in Known High Risk Area', code: '109006' }],
  },
  period: {
    start: {
      value: '2013-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Year,
    },
  },
  status: FlagStatusCode.Inactive,
}

const FlagTwo: Flag = {
  id: '580ca927-34e0-e911-a2c7-005056926fe4|personalertandhazard',
  metadata: mockMetadata,
  author: {
    display: 'John Smith',
    reference:
      'https://phoenixqa.careworks.ie/Services/api/fhir/v3/practitioner/9c1ad2c0-ec34-e911-a2c5-005056926fe4|systemuser',
    type: 'Practitioner',
    identifier: undefined,
  },
  code: {
    coding: [{ display: 'High Risk of Violence', code: '56706' }],
  },
  category: {
    coding: [{ display: 'Violence', code: '66706' }],
  },
  period: {
    start: {
      value: '2013-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
    end: {
      value: '2014-02-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },

  status: FlagStatusCode.Inactive,
}

export { FlagOne, FlagTwo }
