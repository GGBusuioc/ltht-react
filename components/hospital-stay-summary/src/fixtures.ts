import { HospitalStay, PartialDateTimeKindCode, Metadata } from '@ltht-react/types'

const mockMetadata: Metadata = {
  dataSources: [
    {
      display: 'Mock',
    },
  ],
}

const HospitalStayOne: HospitalStay = {
  id: '580ca927-34e0-e911-a2c7-005056926fe4|hospitalstay',
  metadata: mockMetadata,
  nhsNumber: '123456789',
  serviceProvider: { display: 'Chapel Allerton Hospital', type: 'Hospital' },
  period: {
    start: {
      value: '2013-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
    end: {
      value: '2014-02-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
}

const HospitalStayTwo: HospitalStay = {
  id: '690ca927-68e0-c511-b2c7-898056926ze4|hospitalstay',
  metadata: mockMetadata,
  nhsNumber: '987654321',
  serviceProvider: { display: 'Beaumont Hospital', type: 'Hospital' },
  period: {
    start: {
      value: '2020-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
}

export { HospitalStayOne, HospitalStayTwo }
