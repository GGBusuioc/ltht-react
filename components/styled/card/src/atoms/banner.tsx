import { HTMLAttributes, FC } from 'react'
import classNames from 'classnames'
import styled from '@emotion/styled'
import { ChevronIcon } from '@ltht-react/icon'
import { BANNER_COLOURS } from '@ltht-react/styles'

interface StyleProps {
  background: string
  borderBottom: string
  borderTop: string
  color: string
  hover: string
  icon: string
}

const styles = ({ status = 'default' }: Props): StyleProps => {
  switch (status) {
    case 'danger':
      return {
        background: BANNER_COLOURS.DANGER.BACKGROUND,
        borderBottom: BANNER_COLOURS.DANGER.BORDER,
        borderTop: BANNER_COLOURS.DANGER.BORDER,
        color: BANNER_COLOURS.DANGER.TEXT,
        hover: BANNER_COLOURS.DANGER.HOVER,
        icon: BANNER_COLOURS.DANGER.ICON,
      }
    case 'info':
      return {
        background: BANNER_COLOURS.INFO.BACKGROUND,
        borderBottom: BANNER_COLOURS.INFO.BORDER,
        borderTop: BANNER_COLOURS.INFO.BORDER,
        color: BANNER_COLOURS.INFO.TEXT,
        hover: BANNER_COLOURS.INFO.HOVER,
        icon: BANNER_COLOURS.INFO.ICON,
      }
    case 'warning':
      return {
        background: BANNER_COLOURS.WARNING.BACKGROUND,
        borderBottom: BANNER_COLOURS.WARNING.BORDER,
        borderTop: BANNER_COLOURS.WARNING.BORDER,
        color: BANNER_COLOURS.WARNING.TEXT,
        hover: BANNER_COLOURS.WARNING.HOVER,
        icon: BANNER_COLOURS.WARNING.ICON,
      }
    case 'default':
    default:
      return {
        background: BANNER_COLOURS.DEFAULT.BACKGROUND,
        borderBottom: BANNER_COLOURS.DEFAULT.BORDER,
        borderTop: BANNER_COLOURS.DEFAULT.BORDER,
        color: BANNER_COLOURS.DEFAULT.TEXT,
        hover: BANNER_COLOURS.DEFAULT.HOVER,
        icon: BANNER_COLOURS.DEFAULT.ICON,
      }
  }
}

const StyledBanner = styled.div`
  display: flex;
  align-items: center;
  padding: 0.75rem;
  z-index: 1;

  background: ${(props: Props): string => styles(props).background};
  color: ${(props: Props): string => styles(props).color};
  border-top: 1px solid ${(props: Props): string => styles(props).borderTop};
  border-bottom: 1px solid ${(props: Props): string => styles(props).borderBottom};

  & > .card__banner_content {
    flex: 1;
  }

  & > [class^='icon__'] {
    color: ${(props: Props): string => styles(props).icon};
  }

  &:hover {
    background: ${(props: Props): string => (canClick(props) ? styles(props).hover : styles(props).background)};
    cursor: ${(props: Props): string => (canClick(props) ? 'pointer' : 'default')};
  }
`

const Banner: FC<Props> = props => {
  const { classes, children, ...rest } = props
  return (
    <StyledBanner className={classNames(`card__banner-${props?.status || 'default'}`, classes)} {...rest}>
      <div className="card__banner_content">{children}</div>
      {canClick(props) && <ChevronIcon size="medium" direction="right" />}
    </StyledBanner>
  )
}

const canClick = (props: Props): boolean => props.onClick !== undefined

export type StatusTypes = 'default' | 'info' | 'danger' | 'warning'

export interface Props extends HTMLAttributes<HTMLDivElement> {
  classes?: string
  status?: StatusTypes
}

export default Banner
