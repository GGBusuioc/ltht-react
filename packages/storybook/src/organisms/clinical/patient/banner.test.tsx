import React from 'react'
import ReactDOM from 'react-dom'

import PatientBanner from '@ltht-react/patient-banner'
import { DeceasedPatient } from './banner.fixtures'

describe('PatientBanner', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<PatientBanner patient={DeceasedPatient} />, div)
  })
})
