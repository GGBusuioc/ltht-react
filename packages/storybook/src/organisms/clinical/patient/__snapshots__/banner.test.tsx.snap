// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PatientBanner matches snapshot: wrapper mount 1`] = `
.emotion-10 {
  font-family: 'Arial';
  background-color: #231f20;
  border: 0.1rem dashed #f5f7f8;
}

.emotion-9 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  background-color: inherit;
  padding: 0.25rem;
}

@media only screen and (min-width:768px) {
  .emotion-9 {
    -webkit-flex-direction: row;
    -ms-flex-direction: row;
    flex-direction: row;
  }

  .emotion-9 div:first-of-type {
    margin-right: auto;
    border-bottom: none;
  }
}

.emotion-0 {
  color: #fff;
  font-weight: bold;
  font-size: 0.875rem;
}

.emotion-3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.emotion-1 span:first-of-type {
  color: #ccc;
  font-weight: normal;
  font-size: 0.75rem;
}

.emotion-1 span:last-of-type {
  color: #fff;
  font-weight: bold;
  font-size: 0.875rem;
}

.emotion-2 span:first-of-type {
  color: #ccc;
  font-weight: normal;
  font-size: 0.75rem;
}

.emotion-2 span:last-of-type {
  color: #fff;
  font-weight: bold;
  font-size: 0.875rem;
}

.emotion-4 span:first-of-type {
  color: #ccc;
  font-weight: normal;
  font-size: 0.75rem;
}

.emotion-4 span:last-of-type {
  color: #fff;
  font-weight: bold;
  font-size: 0.875rem;
}

.emotion-5 span:first-of-type {
  color: #ccc;
  font-weight: normal;
  font-size: 0.75rem;
}

.emotion-5 span:last-of-type {
  color: #fff;
  font-weight: bold;
  font-size: 0.875rem;
}

.emotion-8 span:first-of-type {
  color: #ccc;
  font-weight: normal;
  font-size: 0.75rem;
}

.emotion-8 span:last-of-type {
  color: #fff;
  font-weight: bold;
  font-size: 0.875rem;
}

.emotion-7 {
  height: 0.6rem;
  width: 0.15rem;
  border-radius: 50%;
  display: inline-block;
  border: 0.15rem solid #fff;
  margin-left: 0.4rem;
  margin-bottom: -0.1rem;
  background-color: #888;
  background-color: #4cc552 !important;
}

@media only screen and (min-width:768px) {
  .emotion-7 {
    margin-bottom: 0;
  }
}

<PatientBanner
  patient={
    Object {
      "active": true,
      "address": Array [
        Object {
          "city": "Leeds",
          "line": Array [
            "18 Benson Gardens",
          ],
          "period": Object {
            "start": Object {
              "kind": "DATE",
              "value": "2020-04-01T00:00:00+00:00",
            },
          },
          "postalCode": "LS11 8HK",
          "text": "18 Benson Gardens, Leeds, LS11 8HK",
          "type": "BOTH",
          "use": "HOME",
        },
      ],
      "birthDate": Object {
        "kind": "DATE",
        "value": "1955-06-30T00:00:00+00:00",
      },
      "deceased": Object {
        "deceasedBoolean": true,
        "deceasedDateTime": Object {
          "kind": "DATE",
          "value": "2020-02-05T00:00:00+00:00",
        },
      },
      "gender": "MALE",
      "id": "829ca260-67e0-e357-b2c7-115087226fg2|patient",
      "identifier": Array [
        Object {
          "extension": Array [
            Object {
              "url": "https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-NHSNumberVerificationStatus-1",
              "valueCodeableConcept": Object {
                "coding": Array [
                  Object {
                    "code": "01",
                    "display": "Number present and verified",
                    "system": "https://fhir.hl7.org.uk/STU3/ValueSet/CareConnect-NHSNumberVerificationStatus-1",
                  },
                ],
              },
              "valueString": "test",
            },
          ],
          "system": "https://fhir.nhs.uk/Id/nhs-number",
          "use": "OFFICIAL",
          "value": "8967452301",
        },
        Object {
          "system": "https://leedsth.nhs.uk/Id/pas-number",
          "use": "OFFICIAL",
          "value": "0056341",
        },
      ],
      "metadata": Object {
        "dataSources": Array [
          Object {
            "display": "Mock Source",
          },
        ],
        "isRedacted": false,
        "requestedWhen": "",
      },
      "name": Array [
        Object {
          "family": "Woo",
          "given": Array [
            "John",
          ],
          "prefix": Array [
            "Mr",
          ],
          "text": "John Woo",
          "use": "USUAL",
        },
      ],
    }
  }
>
  <div
    className="emotion-10"
  >
    <PatientPrimaryInformation
      patient={
        Object {
          "active": true,
          "address": Array [
            Object {
              "city": "Leeds",
              "line": Array [
                "18 Benson Gardens",
              ],
              "period": Object {
                "start": Object {
                  "kind": "DATE",
                  "value": "2020-04-01T00:00:00+00:00",
                },
              },
              "postalCode": "LS11 8HK",
              "text": "18 Benson Gardens, Leeds, LS11 8HK",
              "type": "BOTH",
              "use": "HOME",
            },
          ],
          "birthDate": Object {
            "kind": "DATE",
            "value": "1955-06-30T00:00:00+00:00",
          },
          "deceased": Object {
            "deceasedBoolean": true,
            "deceasedDateTime": Object {
              "kind": "DATE",
              "value": "2020-02-05T00:00:00+00:00",
            },
          },
          "gender": "MALE",
          "id": "829ca260-67e0-e357-b2c7-115087226fg2|patient",
          "identifier": Array [
            Object {
              "extension": Array [
                Object {
                  "url": "https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-NHSNumberVerificationStatus-1",
                  "valueCodeableConcept": Object {
                    "coding": Array [
                      Object {
                        "code": "01",
                        "display": "Number present and verified",
                        "system": "https://fhir.hl7.org.uk/STU3/ValueSet/CareConnect-NHSNumberVerificationStatus-1",
                      },
                    ],
                  },
                  "valueString": "test",
                },
              ],
              "system": "https://fhir.nhs.uk/Id/nhs-number",
              "use": "OFFICIAL",
              "value": "8967452301",
            },
            Object {
              "system": "https://leedsth.nhs.uk/Id/pas-number",
              "use": "OFFICIAL",
              "value": "0056341",
            },
          ],
          "metadata": Object {
            "dataSources": Array [
              Object {
                "display": "Mock Source",
              },
            ],
            "isRedacted": false,
            "requestedWhen": "",
          },
          "name": Array [
            Object {
              "family": "Woo",
              "given": Array [
                "John",
              ],
              "prefix": Array [
                "Mr",
              ],
              "text": "John Woo",
              "use": "USUAL",
            },
          ],
        }
      }
    >
      <div
        className="emotion-9"
      >
        <div>
          <PatientName
            patient={
              Object {
                "active": true,
                "address": Array [
                  Object {
                    "city": "Leeds",
                    "line": Array [
                      "18 Benson Gardens",
                    ],
                    "period": Object {
                      "start": Object {
                        "kind": "DATE",
                        "value": "2020-04-01T00:00:00+00:00",
                      },
                    },
                    "postalCode": "LS11 8HK",
                    "text": "18 Benson Gardens, Leeds, LS11 8HK",
                    "type": "BOTH",
                    "use": "HOME",
                  },
                ],
                "birthDate": Object {
                  "kind": "DATE",
                  "value": "1955-06-30T00:00:00+00:00",
                },
                "deceased": Object {
                  "deceasedBoolean": true,
                  "deceasedDateTime": Object {
                    "kind": "DATE",
                    "value": "2020-02-05T00:00:00+00:00",
                  },
                },
                "gender": "MALE",
                "id": "829ca260-67e0-e357-b2c7-115087226fg2|patient",
                "identifier": Array [
                  Object {
                    "extension": Array [
                      Object {
                        "url": "https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-NHSNumberVerificationStatus-1",
                        "valueCodeableConcept": Object {
                          "coding": Array [
                            Object {
                              "code": "01",
                              "display": "Number present and verified",
                              "system": "https://fhir.hl7.org.uk/STU3/ValueSet/CareConnect-NHSNumberVerificationStatus-1",
                            },
                          ],
                        },
                        "valueString": "test",
                      },
                    ],
                    "system": "https://fhir.nhs.uk/Id/nhs-number",
                    "use": "OFFICIAL",
                    "value": "8967452301",
                  },
                  Object {
                    "system": "https://leedsth.nhs.uk/Id/pas-number",
                    "use": "OFFICIAL",
                    "value": "0056341",
                  },
                ],
                "metadata": Object {
                  "dataSources": Array [
                    Object {
                      "display": "Mock Source",
                    },
                  ],
                  "isRedacted": false,
                  "requestedWhen": "",
                },
                "name": Array [
                  Object {
                    "family": "Woo",
                    "given": Array [
                      "John",
                    ],
                    "prefix": Array [
                      "Mr",
                    ],
                    "text": "John Woo",
                    "use": "USUAL",
                  },
                ],
              }
            }
          >
            <span
              className="emotion-0"
            />
          </PatientName>
        </div>
        <div
          className="emotion-3"
        >
          <DateOfBirth
            patient={
              Object {
                "active": true,
                "address": Array [
                  Object {
                    "city": "Leeds",
                    "line": Array [
                      "18 Benson Gardens",
                    ],
                    "period": Object {
                      "start": Object {
                        "kind": "DATE",
                        "value": "2020-04-01T00:00:00+00:00",
                      },
                    },
                    "postalCode": "LS11 8HK",
                    "text": "18 Benson Gardens, Leeds, LS11 8HK",
                    "type": "BOTH",
                    "use": "HOME",
                  },
                ],
                "birthDate": Object {
                  "kind": "DATE",
                  "value": "1955-06-30T00:00:00+00:00",
                },
                "deceased": Object {
                  "deceasedBoolean": true,
                  "deceasedDateTime": Object {
                    "kind": "DATE",
                    "value": "2020-02-05T00:00:00+00:00",
                  },
                },
                "gender": "MALE",
                "id": "829ca260-67e0-e357-b2c7-115087226fg2|patient",
                "identifier": Array [
                  Object {
                    "extension": Array [
                      Object {
                        "url": "https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-NHSNumberVerificationStatus-1",
                        "valueCodeableConcept": Object {
                          "coding": Array [
                            Object {
                              "code": "01",
                              "display": "Number present and verified",
                              "system": "https://fhir.hl7.org.uk/STU3/ValueSet/CareConnect-NHSNumberVerificationStatus-1",
                            },
                          ],
                        },
                        "valueString": "test",
                      },
                    ],
                    "system": "https://fhir.nhs.uk/Id/nhs-number",
                    "use": "OFFICIAL",
                    "value": "8967452301",
                  },
                  Object {
                    "system": "https://leedsth.nhs.uk/Id/pas-number",
                    "use": "OFFICIAL",
                    "value": "0056341",
                  },
                ],
                "metadata": Object {
                  "dataSources": Array [
                    Object {
                      "display": "Mock Source",
                    },
                  ],
                  "isRedacted": false,
                  "requestedWhen": "",
                },
                "name": Array [
                  Object {
                    "family": "Woo",
                    "given": Array [
                      "John",
                    ],
                    "prefix": Array [
                      "Mr",
                    ],
                    "text": "John Woo",
                    "use": "USUAL",
                  },
                ],
              }
            }
          >
            <div
              className="emotion-1"
            >
              <span>
                Born
              </span>
              <span>
                30-Jun-1955
                 
                25y1955-06-30T00:00:00+00:00
              </span>
            </div>
          </DateOfBirth>
          <DateOfDeath
            patient={
              Object {
                "active": true,
                "address": Array [
                  Object {
                    "city": "Leeds",
                    "line": Array [
                      "18 Benson Gardens",
                    ],
                    "period": Object {
                      "start": Object {
                        "kind": "DATE",
                        "value": "2020-04-01T00:00:00+00:00",
                      },
                    },
                    "postalCode": "LS11 8HK",
                    "text": "18 Benson Gardens, Leeds, LS11 8HK",
                    "type": "BOTH",
                    "use": "HOME",
                  },
                ],
                "birthDate": Object {
                  "kind": "DATE",
                  "value": "1955-06-30T00:00:00+00:00",
                },
                "deceased": Object {
                  "deceasedBoolean": true,
                  "deceasedDateTime": Object {
                    "kind": "DATE",
                    "value": "2020-02-05T00:00:00+00:00",
                  },
                },
                "gender": "MALE",
                "id": "829ca260-67e0-e357-b2c7-115087226fg2|patient",
                "identifier": Array [
                  Object {
                    "extension": Array [
                      Object {
                        "url": "https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-NHSNumberVerificationStatus-1",
                        "valueCodeableConcept": Object {
                          "coding": Array [
                            Object {
                              "code": "01",
                              "display": "Number present and verified",
                              "system": "https://fhir.hl7.org.uk/STU3/ValueSet/CareConnect-NHSNumberVerificationStatus-1",
                            },
                          ],
                        },
                        "valueString": "test",
                      },
                    ],
                    "system": "https://fhir.nhs.uk/Id/nhs-number",
                    "use": "OFFICIAL",
                    "value": "8967452301",
                  },
                  Object {
                    "system": "https://leedsth.nhs.uk/Id/pas-number",
                    "use": "OFFICIAL",
                    "value": "0056341",
                  },
                ],
                "metadata": Object {
                  "dataSources": Array [
                    Object {
                      "display": "Mock Source",
                    },
                  ],
                  "isRedacted": false,
                  "requestedWhen": "",
                },
                "name": Array [
                  Object {
                    "family": "Woo",
                    "given": Array [
                      "John",
                    ],
                    "prefix": Array [
                      "Mr",
                    ],
                    "text": "John Woo",
                    "use": "USUAL",
                  },
                ],
              }
            }
          >
            <div
              className="emotion-2"
            >
              <span>
                Died
              </span>
              <span>
                05-Feb-2020
              </span>
            </div>
          </DateOfDeath>
        </div>
        <div
          className="emotion-3"
        >
          <Gender
            patient={
              Object {
                "active": true,
                "address": Array [
                  Object {
                    "city": "Leeds",
                    "line": Array [
                      "18 Benson Gardens",
                    ],
                    "period": Object {
                      "start": Object {
                        "kind": "DATE",
                        "value": "2020-04-01T00:00:00+00:00",
                      },
                    },
                    "postalCode": "LS11 8HK",
                    "text": "18 Benson Gardens, Leeds, LS11 8HK",
                    "type": "BOTH",
                    "use": "HOME",
                  },
                ],
                "birthDate": Object {
                  "kind": "DATE",
                  "value": "1955-06-30T00:00:00+00:00",
                },
                "deceased": Object {
                  "deceasedBoolean": true,
                  "deceasedDateTime": Object {
                    "kind": "DATE",
                    "value": "2020-02-05T00:00:00+00:00",
                  },
                },
                "gender": "MALE",
                "id": "829ca260-67e0-e357-b2c7-115087226fg2|patient",
                "identifier": Array [
                  Object {
                    "extension": Array [
                      Object {
                        "url": "https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-NHSNumberVerificationStatus-1",
                        "valueCodeableConcept": Object {
                          "coding": Array [
                            Object {
                              "code": "01",
                              "display": "Number present and verified",
                              "system": "https://fhir.hl7.org.uk/STU3/ValueSet/CareConnect-NHSNumberVerificationStatus-1",
                            },
                          ],
                        },
                        "valueString": "test",
                      },
                    ],
                    "system": "https://fhir.nhs.uk/Id/nhs-number",
                    "use": "OFFICIAL",
                    "value": "8967452301",
                  },
                  Object {
                    "system": "https://leedsth.nhs.uk/Id/pas-number",
                    "use": "OFFICIAL",
                    "value": "0056341",
                  },
                ],
                "metadata": Object {
                  "dataSources": Array [
                    Object {
                      "display": "Mock Source",
                    },
                  ],
                  "isRedacted": false,
                  "requestedWhen": "",
                },
                "name": Array [
                  Object {
                    "family": "Woo",
                    "given": Array [
                      "John",
                    ],
                    "prefix": Array [
                      "Mr",
                    ],
                    "text": "John Woo",
                    "use": "USUAL",
                  },
                ],
              }
            }
          >
            <div
              className="emotion-4"
            >
              <span>
                Gender
              </span>
              <span>
                Male
              </span>
            </div>
          </Gender>
          <PatientGender
            patient={
              Object {
                "active": true,
                "address": Array [
                  Object {
                    "city": "Leeds",
                    "line": Array [
                      "18 Benson Gardens",
                    ],
                    "period": Object {
                      "start": Object {
                        "kind": "DATE",
                        "value": "2020-04-01T00:00:00+00:00",
                      },
                    },
                    "postalCode": "LS11 8HK",
                    "text": "18 Benson Gardens, Leeds, LS11 8HK",
                    "type": "BOTH",
                    "use": "HOME",
                  },
                ],
                "birthDate": Object {
                  "kind": "DATE",
                  "value": "1955-06-30T00:00:00+00:00",
                },
                "deceased": Object {
                  "deceasedBoolean": true,
                  "deceasedDateTime": Object {
                    "kind": "DATE",
                    "value": "2020-02-05T00:00:00+00:00",
                  },
                },
                "gender": "MALE",
                "id": "829ca260-67e0-e357-b2c7-115087226fg2|patient",
                "identifier": Array [
                  Object {
                    "extension": Array [
                      Object {
                        "url": "https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-NHSNumberVerificationStatus-1",
                        "valueCodeableConcept": Object {
                          "coding": Array [
                            Object {
                              "code": "01",
                              "display": "Number present and verified",
                              "system": "https://fhir.hl7.org.uk/STU3/ValueSet/CareConnect-NHSNumberVerificationStatus-1",
                            },
                          ],
                        },
                        "valueString": "test",
                      },
                    ],
                    "system": "https://fhir.nhs.uk/Id/nhs-number",
                    "use": "OFFICIAL",
                    "value": "8967452301",
                  },
                  Object {
                    "system": "https://leedsth.nhs.uk/Id/pas-number",
                    "use": "OFFICIAL",
                    "value": "0056341",
                  },
                ],
                "metadata": Object {
                  "dataSources": Array [
                    Object {
                      "display": "Mock Source",
                    },
                  ],
                  "isRedacted": false,
                  "requestedWhen": "",
                },
                "name": Array [
                  Object {
                    "family": "Woo",
                    "given": Array [
                      "John",
                    ],
                    "prefix": Array [
                      "Mr",
                    ],
                    "text": "John Woo",
                    "use": "USUAL",
                  },
                ],
              }
            }
          >
            <div
              className="emotion-5"
            >
              <span>
                Age at Death
              </span>
              <span>
                25y1955-06-30T00:00:00+00:00
              </span>
            </div>
          </PatientGender>
        </div>
        <div>
          <NhsNumber
            patient={
              Object {
                "active": true,
                "address": Array [
                  Object {
                    "city": "Leeds",
                    "line": Array [
                      "18 Benson Gardens",
                    ],
                    "period": Object {
                      "start": Object {
                        "kind": "DATE",
                        "value": "2020-04-01T00:00:00+00:00",
                      },
                    },
                    "postalCode": "LS11 8HK",
                    "text": "18 Benson Gardens, Leeds, LS11 8HK",
                    "type": "BOTH",
                    "use": "HOME",
                  },
                ],
                "birthDate": Object {
                  "kind": "DATE",
                  "value": "1955-06-30T00:00:00+00:00",
                },
                "deceased": Object {
                  "deceasedBoolean": true,
                  "deceasedDateTime": Object {
                    "kind": "DATE",
                    "value": "2020-02-05T00:00:00+00:00",
                  },
                },
                "gender": "MALE",
                "id": "829ca260-67e0-e357-b2c7-115087226fg2|patient",
                "identifier": Array [
                  Object {
                    "extension": Array [
                      Object {
                        "url": "https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-NHSNumberVerificationStatus-1",
                        "valueCodeableConcept": Object {
                          "coding": Array [
                            Object {
                              "code": "01",
                              "display": "Number present and verified",
                              "system": "https://fhir.hl7.org.uk/STU3/ValueSet/CareConnect-NHSNumberVerificationStatus-1",
                            },
                          ],
                        },
                        "valueString": "test",
                      },
                    ],
                    "system": "https://fhir.nhs.uk/Id/nhs-number",
                    "use": "OFFICIAL",
                    "value": "8967452301",
                  },
                  Object {
                    "system": "https://leedsth.nhs.uk/Id/pas-number",
                    "use": "OFFICIAL",
                    "value": "0056341",
                  },
                ],
                "metadata": Object {
                  "dataSources": Array [
                    Object {
                      "display": "Mock Source",
                    },
                  ],
                  "isRedacted": false,
                  "requestedWhen": "",
                },
                "name": Array [
                  Object {
                    "family": "Woo",
                    "given": Array [
                      "John",
                    ],
                    "prefix": Array [
                      "Mr",
                    ],
                    "text": "John Woo",
                    "use": "USUAL",
                  },
                ],
              }
            }
          >
            <div
              className="emotion-8"
            >
              <span>
                NHS No.
              </span>
              <span
                className="emotion-7"
                title="Verified"
              />
              <span>
                896 745 2301
              </span>
            </div>
          </NhsNumber>
        </div>
      </div>
    </PatientPrimaryInformation>
  </div>
</PatientBanner>
`;
